ompi/mca/sharedfp/lockedfile/sharedfp_lockedfile_file_open.c

```diff
@@ -112,8 +115,27 @@ int mca_sharedfp_lockedfile_file_open (struct ompi_communicator_t *comm,
         return OMPI_ERR_OUT_OF_RESOURCE;
     }
     snprintf(lockedfilename, filenamelen, "%s-%u-%d%s",filename,masterjobid,int_pid,".lock");
-    module_data->filename = lockedfilename;
-    
+    if (opal_path_is_absolute(lockedfilename) ) {
+        module_data->filename = lockedfilename;
+    }
```

Comment:
OMPI standard is really `} else {` all on one line.

Reply:
done (but to be fair there are tons of places at least in my code where it is done in this format)